packages/back-end/src/controllers/experiments.ts

```diff
@@ -297,14 +298,24 @@ export async function getExperiment(
     experiment.id
   );
 
-  const linkedFeatures = await getLinkedFeatureInfo(context, experiment);
+  const linkedFeatureInfo = await getLinkedFeatureInfo(context, experiment);
+
+  const linkedFeatureIds = experiment.linkedFeatures || [];
+
+  const linkedFeatures = await getFeaturesByIds(context, linkedFeatureIds);
```

Comment:
We should add an early return to the `getFeaturesByIds` method if the array is empty (right now, it will still run the Mongo query).